src/main/java/org/folio/pv/service/ValidationRuleServiceImpl.java

```diff
@@ -109,6 +113,15 @@ public ValidationResult validatePasswordByRules(final Password passwordContainer
     return validationResult;
   }
 
+  private void validatePasswordContainer(Password passwordContainer) {
```

Comment:
There is no need for this validation. Please, add hibernate-validator dependency to the project and handle exceptions that this dependency throw.

Reply:
Also as we discussed if request body validation failed - return 422 code.

Reply:
Added spring-boot-starter-validation dependency.
Catching `MethodArgumentNotValidException`
and building response body with 422 status code